HidPkg/Include/Protocol/HidIo.h

```diff
@@ -0,0 +1,160 @@
+/** @file
+  HID I/O
+
+  This protocol is used by code, typically drivers, running in the EFI
+  boot services environment to access HID devices.
+
+  Copyright (c) Microsoft Corporation. All rights reserved.
+

```

Comment:
Need the SPDX identifier.

Reply:
This is true for many of the new files, not going to leave additional comments on them.

Reply:
Fixed.